Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Last-Update: 2012-02-10
Forwarded: yes
Description: fixes FTBFS with format hardening
 Workaround for error: "format not a string literal and
 "no format arguments [-Werror=format-security]"

--- a/wvRTF.c
+++ b/wvRTF.c
@@ -192,7 +192,7 @@
     rtf_output_char ('{');
 
     /* font color */
-    rtf_output (rtfColors[ud->cCol]);
+    rtf_output ("%s", rtfColors[ud->cCol]);
 
     /* font face */
     rtf_output ("\\f%d", ud->cFont);
